precode.go

```diff
@@ -40,13 +41,80 @@ var tasks = map[string]Task{
 }
 
 // Ниже напишите обработчики для каждого эндпоинта
-// ...
+func getTasksHandler(w http.ResponseWriter, r *http.Request) {
+	res := make([]Task, 0, len(tasks))
+
+	for _, task := range tasks {
+		res = append(res, Task{
+			ID:           task.ID,
+			Description:  task.Description,
+			Note:         task.Note,
+			Applications: task.Applications,
+		})
+	}
+
+	out, err := json.Marshal(&res)
```

Comment:
Копирование тасок бессмысленно в данном контексте, можно сразу их вернуть

```suggestion
	out, err := json.Marshal(&task)
```